components/ai_chat/resources/page/components/attachment_item/index.tsx

```diff
@@ -34,6 +34,16 @@ export const formatFileSize = (bytes: number): string => {
   return `${size.toFixed(2)} ${units[index]}`
 }
 
+/**
+ * Checks if a file is a full page screenshot
+ * @param file - The uploaded file to check
+ * @returns True if the file is a full page screenshot
+ */
+const isFullPageScreenshot = (file: Mojom.UploadedFile): boolean => {
```

Comment:
Maybe we should move this function into `conversation_context` and export it rather than reimplementing it there?

Reply:
addressed that in the latest commit